ImageNet/inference/Alipay_resnet101_npu_tensorflow.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "Ai CognitiveComputing team in Alipay Group",
+    "authorEmail": "xing.rao@alibaba-inc.com, weihong.wh@alibaba-inc.com",
+    "framework": "tensorflow+NpuInference",
+    "codeURL": "https://github.com/alipay-cognition/DAWNBench_Inference",
+    "commitHash": "",
```

Please don't leave the `commitHash` field blank. Assuming you want the most recent commit on master, it should be:

```suggestion
    "commitHash": "ea76aaa2699e2dd1fdf46ae144c332f83f4b3d9c",
```